README.md

```diff
@@ -1,48 +1,36 @@
 [![Python Version](https://img.shields.io/badge/python-3.9+-blue.svg)](https://www.python.org/)
-[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
 # TDP Lib
 
-TDP lib is a [Python](https://www.python.org/) library built on top of [Ansible](https://www.ansible.com/) to manage clusters. It provides a set of tools to overcome the limitations of Ansible, such as:
+`tdp-lib` is a [Python](https://www.python.org/) library built on top of [Ansible](https://www.ansible.com/). It is designed for managing TDP clusters, offering advanced tools to extend Ansible's capabilities. Key features include:
 
-- Defining a DAG of tasks based on relations between services and components
-- Defining variables in a single place
+- Creating a Directed Acyclic Graph (DAG) of tasks to manage dependencies and relationships between services and components.
+- Centralized configuration management through a unified variable definition system.
 
-`tdp-lib` can be used as a Python library, through an admin [CLI](#cli-usage), a REST API (see [`tdp-server`](https://github.com/tOSIT-IO/tdp-server)) or a web interface (see [`tdp-ui`](https://github.com/tOSIT-IO/tdp-ui)).
+`tdp-lib` can be utilized in various ways: as a Python library, through an admin [CLI](#cli-usage), via a REST API (see [`tdp-server`](https://github.com/TOSIT-IO/tdp-server)), or through a web interface (see [`tdp-ui`](https://github.com/TOSIT-IO/tdp-ui)).
 
 ## Pre-requisites
 
-TDP lib requires:
+To use `tdp-lib`, ensure you have the following prerequisites:
 
-- Python 3.9+ with [Poetry](https://python-poetry.org/)
-- A RDBMS system (such as [PostgreSQL](https://www.postgresql.org/) or [SQLite](https://www.sqlite.org/index.html))
+- Python 3.9 or higher.
```

Comment:
We should say that we test only python 3.9 to avoid issue about python 3.10+ if something does not work.
```suggestion
- Python 3.9 or higher. Higher version are not guarantee to work, only 3.9 is tested.
```